package/MDAnalysis/core/selection.py

```diff
@@ -655,8 +655,16 @@ def _apply(self, group):
         pattern = Chem.MolFromSmarts(self.pattern)
         if not pattern:
             raise ValueError(f"{self.pattern!r} is not a valid SMARTS query")
+        if "maxMatches" in self.rdkit_kwargs:
```

Comment:
Not sure if it's somewhere else, but maybe we should document the rdkit_kwargs option if it isn't already?

Reply:
I added documentation to `select_atoms`, is there anywhere else this should be documented?